posts/2023-07-20_cross_company_dev/cross_industry_dev.qmd

```diff
@@ -0,0 +1,148 @@
+If you would like to learn more about licenses for open source projects in the clinical reporting world please check out the [recent E2E guidance on open source license](https://phuse-org.github.io/E2E-OS-Guidance/) 
+
+## Development workflow
+
+Every improvement, task, or feature we want to implement on [{admiral}](https://github.com/pharmaverse/admiral) starts as an issue on our [GitHub repository](https://github.com/pharmaverse/admiral).
+It is the centerpiece of our development workflow, along with our [developer guides](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) which describe in detail the strategies, conventions, and workflows used in development.
+The guides help us keep the [{admiral}](https://github.com/pharmaverse/admiral) package internally consistent (e.g. naming conventions, function logic) but also ensure that [{admiral}](https://github.com/pharmaverse/admiral) adjacent packages follow the same conventions and share the user interface.
+This is further helped by the implemented CICD pipeline which ensures styling convention and spelling (and much more).
+
+
+The core package developers team meets once a week (twice a week before a release) to discuss progress and priorities. 
+Here, the role of product lead (currently [Edoardo Mancini](https://www.linkedin.com/in/edoardo-mancini-7a4a40148/) at Roche) and technical lead (currently [Ben Straub](https://www.linkedin.com/in/ben-straub/) at GSK) is to set priorities and track the release schedule.
+These stand-up meetings are centered around the [project-board](https://github.com/orgs/pharmaverse/projects/12) which gives a complete overview of activities and progress.
+Issues are mostly self-assigned so developers can really chose what they want to work on. 
+
+![GitHub project board](project_board.png)
+
+By design, [{admiral}](https://github.com/pharmaverse/admiral) is community built. 
+Most developers working on the project are statistical programmers working on clinical reporting themselves. 
+As an open source project, community input is highly valued, and anyone using [{admiral}](https://github.com/pharmaverse/admiral) is encouraged to submit issues or take on issues as part of the development team.
+We also do occasional events to bring the statistical programmers community and the developers closer together. 
+Just last February we organised the [{admiral}](https://github.com/pharmaverse/admiral) hackathon which had up to 500 participants.
+
+
+## Impact
+
+For Roche, cross-industry package development work out in their favor: They get access to software created by specialists and users from across the industry but paying only a fraction of the developmental costs. 
+Of course, they don't have total developmental control but they do get a seat at the table. 
+Any gaps between the open source [{admiral}](https://github.com/pharmaverse/admiral) package and the proprietary Roche workflow were bridged by the internal {admiralRoche} package. 
```

Comment:
@StefanThoma minor but we avoided upper case in package names so its {admiralroche}